### ISSUE TITLE
Contract compilation doesn't work when kernel is running on Ubuntu 18.04.1 LTS

### ISSUE THREAD

**Issue by satran004** [aionnetwork/aion#628]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Aion version?**: Aion v0.3.1 - Mainnet - Moldoveanu Peak
> - **Which operating system?**: Linux
> - **Which distributor and version?**: Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-32-generic x86_64)
> - **How installed?**: via binaries 
> - **Are you fully synchronized?**: yes
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

I am trying to compile a solidity contract through java api. But the compilation fails with  a response code -4.
Result: ERROR BSE  [api]: [validRspHeader] Failed return code: [-4]

Java API : ITx.compile

The compilation is also failing with web3js.

But the same client code works with kernel running on Ubuntu 16.04.

**Comment by qoire** [comment]
@satran004 Hey, as a sanity check for us can I get you to navigate from the Aion home folder to ``native/linux/solidity`` and execute solidity by doing: ``./solidity`` within that folder? Can you paste the output, thanks.

**Comment by yulongaion** [comment]
@qoire the `solc` will work with Ubuntu 18.04 after #624
